src/pal/src/map/virtual.cpp

```diff
@@ -1073,6 +1073,8 @@ VIRTUALCommitMemory(
     nProtect = W32toUnixAccessControl(flProtect);
     vProtect = VIRTUALConvertWinFlags(flProtect);
 
+    StartBoundary = pInformation->startBoundary + runStart * VIRTUAL_PAGE_SIZE;
```

Comment:
@janvorli i have initialized pRetVal here too as there is another jump to error condition below